src/Control/Eff/QuickStart.hs

```diff
@@ -45,6 +45,13 @@ import           Control.Eff.State.Lazy
 import           Control.Eff.Exception
 import           Control.Monad                            ( when )
 
+-- | a function that runs through some examples
+quickstart :: IO ()
+quickstart = do
```

Comment:
@power-fungus what's the purpose of this function? i don't quite understand the intent here

Reply:
@suhailshergill It mainly has the purpose that the quickstart module can be linked at haddock-level in the documentation of `Eff`. Maybe I will be able to fill it some day with a useful interactive tutorial.

It can be deleted if the cross-reference to the QuickStart module is not necessary.